From ad136902aab9e0bfd86e52e3879f78ceac85055c Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Fri, 16 May 2014 18:49:19 +0300
Subject: [PATCH] JVM/Exceptions.hs: import Data.Typeable to fix build failure
 on ghc-7.8
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Looks like Instances stopped leaking-in this file:
> [ 7 of 20] Compiling JVM.Exceptions   ( JVM/Exceptions.hs, dist/build/JVM/Exceptions.o )
>
> JVM/Exceptions.hs:10:13:
>     Not in scope: type constructor or class ‘Typeable’
>     Perhaps you meant ‘Typeable1’ (imported from Control.Monad.Exception)

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 JVM/Exceptions.hs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/JVM/Exceptions.hs b/JVM/Exceptions.hs
index 2d47af4..09283a6 100644
--- a/JVM/Exceptions.hs
+++ b/JVM/Exceptions.hs
@@ -3,6 +3,7 @@ module JVM.Exceptions where
 
 import Control.Monad.Exception
 import qualified Data.ByteString.Lazy as B
+import Data.Typeable (Typeable)
 
 import JVM.ClassFile
 
-- 
1.9.2

